tests/tests.js

```diff
@@ -1,33 +1,157 @@
-const { expect } = require('chai')
-const { describe, it } = require('mocha')
+const chai = require('chai')
+const sinon = require('sinon')
+const sinonChai = require('sinon-chai')
+const {
+  afterEach,
+  beforeEach,
+  describe,
+  it
+} = require('mocha')
 const employee = require('./employee')
 const products = require('./products')
 const pricing = require('../pricing')
 
-describe('calculateProductPrice', () => {
-  it('returns the price for a voluntary life product for a single employee', () => {
+chai.use(sinonChai)
+const {
+  expect
+} = chai
+
+describe('formatPrice', () => {
+  let spyOnPrice
+  let spyOnformatPrice
+  let sandbox
+
+  foreEach(() => {
+    sandbox = sinon.createSandbox()
+
+    priceSpy = sandbox.spy(pricing, 'price')
+    formatPriceSpy = sandbox.spy(pricing, 'formatPrice')
+  })
+  afterEach(() => {
+    sandbox.restore()
+    it('returns a truncated price with two decimal places not rounded when given a number with multiple places', () => {
+      const x = 15.1566663
+      const formattedPrice = pricing.formatPrice(15.1566663)
+
+      expect(formattedPrice).to.equal(15.15)
+    })
```

Comment:
You should never have a test in your `afterEach`, you are now running this test after each other test.